free-programming-books.md

```diff
@@ -565,6 +565,7 @@
 * [Seven Sketches in Compositionality: An Invitation to Applied Category Theory](https://arxiv.org/pdf/1803.05316.pdf) - Brendan Fong and David I. Spivak (PDF)
 * [Statistical Thinking for the 21st Century](https://statsthinking21.org) - Russell A. Poldrack
 * [Statistics Done Wrong](http://www.statisticsdonewrong.com) - Alex Reinhart
+* [The Joy of Cryptography (2020)](https://web.engr.oregonstate.edu/~rosulekm/crypto/crypto.pdf) - Mike Rosulek (PDF)
```

Comment:
So excited about this textbook! Could you use https://web.engr.oregonstate.edu/~rosulekm/crypto/ as the link, and add the "In progress" notation?

Reply:
@eshellman I added the link and put the in process notation. Thank you for your review!